Formula/katago.rb

```diff
@@ -0,0 +1,36 @@
+class Katago < Formula
+  desc "Neural Network Go engine with no human-provided knowledge"
+  homepage "https://github.com/lightvector/KataGo"
+  url "https://github.com/lightvector/KataGo/archive/v1.3.2.tar.gz"
+  sha256 "75e783cd2db6180f2cc694703df809e5c04ba24c09546a9c24aad575e18ef2d8"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "libzip"
+
+  resource "15b-network" do
+    url "https://github.com/lightvector/KataGo/releases/download/v1.3.2/g170e-b15c192-s1672170752-d466197061.txt.gz", :using => :nounzip
+    sha256 "09456899f1b9155217f4ca059cf8a68f79b7eba22cf6b7c00ffb7f17ce067967"
+  end
+
+  resource "20b-network" do
+    url "https://github.com/lightvector/KataGo/releases/download/v1.3.2/g170-b20c256x2-s1913382912-d435450331.txt.gz", :using => :nounzip
+    sha256 "c309c0361ca1109084dd026235c9fa0254ffe22eecf096b6c5988a9d902a75ca"
+  end
+
+  def install
+    cd "cpp" do
+      system "cmake", ".", "-DBUILD_MCTS=1", "-DUSE_BACKEND=OPENCL", "-DNO_GIT_REVISION=1", "-DCMAKE_OSX_SYSROOT=#{MacOS.sdk_path}", *std_cmake_args
+      system "make"
+      bin.install "katago"
+      (etc / "katago").install "configs"
```

Comment:
@apetresc I think that this should maybe put into the pkgshare directory too, as a) these are updated regularly and are considered part of the documentation and b) are not listed when executing `brew ls katago`, so it is harder to discover them for users, especially if there is no output during installation that tells users about them. Therefore I think we should use `pkgshare.install "configs"` instead.

Reply:
People will probably not run the runtests command regularly, except when they see problems.

However, the code exercised by the tests is the basic math code that is executed all the time when actually loading and executing neural networks to play go. So I think this is a good compromise to check that the compile was correct.

It would be possible to build a true end to end test, that analyses a SGF file or tries to play a game of go via the gpt protocol, but I think that would be overkill here (for now).

I am in contact with the developers, and he suggested that the tests executed by `runtests` would be a good basis. See https://github.com/lightvector/KataGo/issues/112#issuecomment-580552392 for details.

Do you require additional tests for this to be merged?

Reply:
Alright, you've convinced me. Especially since the configs are all samples, they act more as documentation than Unix-style `/etc` entries. I've moved them now :)